### PR TITLE
Add footer (SHOWROOM-169)

### PR DESCRIPTION
**Pull request by paszkowskiDamian**

### CODE REVIEW

File: source/theme/landing-page-footer.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<footer class="section landing-page-footer">
+  <ul class="section-wrapper">
+    <li class="footer__navigation-group">
+      <a class="footer__navigation-link">Guides</a>
+      <ul>
+        <li><a class="footer__navigation-link">Article one two</a></li>
```
Comment (paszkowskiDamian) [inline source/theme/landing-page-footer.html]: Looks like this navigation should also be generated automatically based on content.

**Comment by paszkowskiDamian** [comment]
image

What is going on here?

**Comment by NataliiaKononikhina** [comment]
> image
> 
> What is going on here?

It was fixed in another PR.

File: source/theme/landing-page-footer.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        <li><a class="footer__navigation-link">And article</a></li>
+        <li><a class="footer__navigation-link">Three</a></li>
+        <li><a class="footer__navigation-link">Article four</a></li>
+        <li><a class="footer__navigation-link">Another</a></li>
+        <li><a class="footer__navigation-link">Six</a></li>
+      </ul>
+    </li>
+    <li class="footer__navigation-group">
```
Comment (paszkowskiDamian) [inline source/theme/landing-page-footer.html]: We could make a macro out of it.

**Pull request merged**